docs/quicktips/multi-cam.md

```diff
@@ -12,75 +12,86 @@ description: >-
+## Edit the original configuration file for a specific camera
+Find the path to the camera by its ID. This makes it easier to stop `dev/video0` and `/dev/video1` from being switchedin the future.
 
-Run `ls /dev/v4l/by-id`, copy your camera id and paste it into the editor in mainsail. For example:
+In a termina, run the comman `ls /dev/v4l/by-id`, copy your camera id and paste it into the editor in Mainsail. For example:
 
 ```bash
 $ ls  /dev/v4l/by-id/
+```
+
+returns
+
+```
 usb-046d_0825_88C56B60-video-index0  usb-046d_0825_88C56B60-video-index1
 ```
-I'll use the first one `usb-046d_0825_88C56B60-video-index0`
+This example will use the first one: `usb-046d_0825_88C56B60-video-index0`
 
 
 __Note__  
 Not sure which device is which? You can run lsusb to match a name to an ID.
 {: .info}
 
-Then edit `/home/pi/klipper_config/webcam.txt` in by clicking on it in mainsail.
+Edit `/home/pi/klipper_config/webcam.txt`  by clicking on it in Mainsail.
 
-Set `camera="usb"` at the top, and the `camera_usb_options` line to indicate the device:
+Add `camera="usb"` at the beginning of the file and then add a line with  `camera_usb_options` to indicate the device:
 
 `camera_usb_options="-r 640x480 -f 10 -d /dev/v4l/by-id/<device long id>"`
 
 __Note__  
-Using a Raspberry Pi camera? Set camera="raspi" at the top of the file, and you can ignore referencing the cameras by device ID.
+If you are using a Raspberry Pi branded camera, seet camera="raspi" at the top of the file and you do not have to reference the cameras by device ID.
+
 {: .info}
 
 ## 3. Editing the second configuration file
-There's two things that need adjusting here: the port mjpg_streamer will be running under, and the device again.
+When using a second camera, you must add the port `mjpg_streamer` runs under and the device.
 
 __Note__  
-Using 1 RPi camera and 1 USB camera? You don't need to reference them by ID, since camera="raspi" vs. camera="usb" is enough.
+If you are using one Raspberry Pi branded camera and a different USB camera you don't need to reference them by ID, as camera="raspi" and camera="usb" is enough.
 {: .info}
 
-Find the long ID of the second camera, as you did for the first one.
+Similar to finding the first camera, run `ls` and copy the ID of the second camera:
 
-Editing `/home/pi/klipper_config/webcam2.txt` like described above.
+```bash
+ls  /dev/v4l/by-id/
+```
 
-As above again, set camera="usb" and set camera_usb_options but this time using the ID for the second camera:
+Edit `/home/pi/klipper_config/webcam2.txt` as described above andd add:
+`camera="usb"` and `set camera_usb_options` but this time using the ID for the second camera:
 
-`camera_usb_options="-r 640x480 -f 10 -d /dev/v4l/by-id/<device long id>"`
+```bash
+camera_usb_options="-r 640x480 -f 10 -d /dev/v4l/by-id/<device long id>"
+```
 
-Adjust the port, by editing camera_http_options:
+Adjust the port by editing `camera_http_options`:
```

Comment:
just as refererence ("add"), this option is also included and commented out in the file:
https://github.com/mainsail-crew/MainsailOS/blob/f622e1ce6ebd3e6c69d632ebce4ce709669571bd/src/modules/mjpgstreamer/filesystem/home/pi/klipper_config/webcam.txt#L72